app/controllers/devise/displayqr_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Devise::DisplayqrController < DeviseController
+  layout 'application'
```

Comment:
@soniankatha, Load the layout dynamically since 'application' layout is for users.